### PR TITLE
Add SectionHeadline component

### PR DESCRIPTION
**Pull request by BelfDev**
# SectionHeadline component

**Mobile**
Screen Shot 2020-03-16 at 2 01 31 AM

**Desktop**
Screen Shot 2020-03-16 at 2 01 41 AM

- Create `SectionHeadline` component
- Add proper styles
- Adjust typography font sizes
- Format `SocialMediaBar` query properly

### CODE REVIEW

File: src/components/common/texts/SectionHeadline/SetionHeadline.module.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+@import "src/resources/styles/breakpoints";
+
+.sectionHeadline {
+  display: block;
+
+  .pre-headline {
+    margin: 0 0 4px;
+    color: $color-neutral-dark-gray;
+    text-align: left;
+    font-weight: 400;
```
Comment (yagotome) [inline src/components/common/texts/SectionHeadline/SetionHeadline.module.scss]:
```suggestion
    font-weight: normal;
```

File: src/components/common/texts/SectionHeadline/SetionHeadline.module.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    font-size: $typography-mobile-h3;
+
+    @include desktop {
+      margin: 0 0 12px;
```
Comment (yagotome) [inline src/components/common/texts/SectionHeadline/SetionHeadline.module.scss]:
```suggestion
      margin-bottom: 12px;
```

File: src/components/common/texts/SectionHeadline/SetionHeadline.module.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+.sectionHeadline {
+  display: block;
+
+  .pre-headline {
+    margin: 0 0 4px;
+    color: $color-neutral-dark-gray;
+    text-align: left;
+    font-weight: 400;
+    font-size: $typography-mobile-h3;
```
Comment (yagotome) [inline src/components/common/texts/SectionHeadline/SetionHeadline.module.scss]:
```suggestion
    font-size: $typography-mobile-h3;
    line-height: $typography-mobile-h3;
```

File: src/components/pages/HomePage/HomePage.js [hunk 4]
```diff
@@ -30,6 +31,12 @@ const HomePage = ({ data: { cms } }) => {
 
       <FAB className={styles.fab} />
 
+      <SectionHeadline
+        className={styles.sectionHeadline}
+        preHeadline={'como'}
+        headline={'Fazemos'}
```
Comment (yagotome) [inline src/components/pages/HomePage/HomePage.js]: It's missing translations, isn't it?

```suggestion
        preHeadline={t('how')}
        headline={t('we_do')}
```

File: src/components/pages/HomePage/HomePage.module.scss [hunk 5]
```diff
@@ -15,3 +15,8 @@
   bottom: 16px;
   z-index: 1;
 }
+
+.sectionHeadline {
+  display: block;
+  margin: 32px auto 32px;
```
Comment (yagotome) [inline src/components/pages/HomePage/HomePage.module.scss]:
```suggestion
  margin: 32px auto;
```

File: src/components/common/texts/SectionHeadline/SectionHeadline.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+SectionHeadline.propTypes = {
+  className: PropTypes.string,
+  preHeadline: PropTypes.string.isRequired,
+  headline: PropTypes.string.isRequired,
+};
+
+SectionHeadline.defaultProps = {
+  className: '',
+  preHeadline: '',
+  headline: '',
```
Comment (yagotome) [inline src/components/common/texts/SectionHeadline/SectionHeadline.js]: Required fields don't need default values.

```suggestion
```

Comment (BelfDev) [reply src/components/common/texts/SectionHeadline/SectionHeadline.js]: Thanks for pointing this out. I set these fields as required after I created the default values. Fixed.

File: src/components/common/texts/SectionHeadline/SetionHeadline.module.scss (hunk 1 above)
Comment (BelfDev) [reply src/components/common/texts/SectionHeadline/SetionHeadline.module.scss]: Disagreed. Below are the font-weight options that we have. They are declared in [typographt.js](https://github.com/lutick/website/blob/cfbe55176c50f77ce3d4f11b687076397faf8845/src/utils/typography.js).
`styles: ['100', '200', '300', '400', '500', '600', '700', '800', '900']`

Using 400 makes it absolutely **clear** and **consistent.**

File: src/components/common/texts/SectionHeadline/SetionHeadline.module.scss (hunk 3 above)
Comment (BelfDev) [reply src/components/common/texts/SectionHeadline/SetionHeadline.module.scss]: Accepted. I noticed this property had an effect on the `firefox` browser.

File: src/components/common/texts/SectionHeadline/SetionHeadline.module.scss (hunk 2 above)
Comment (BelfDev) [reply src/components/common/texts/SectionHeadline/SetionHeadline.module.scss]: ...

File: src/components/pages/HomePage/HomePage.js (hunk 4 above)
Comment (BelfDev) [reply src/components/pages/HomePage/HomePage.js]: Yes. But again, it was not my focus. I prefer to build the components first and wire up the data later when I'm finishing up the `Page` layout.

File: src/components/pages/HomePage/HomePage.module.scss (hunk 5 above)
Comment (BelfDev) [reply src/components/pages/HomePage/HomePage.module.scss]: This is just for showcasing purposes.

**Pull request merged**